fe/fe-core/src/main/java/com/starrocks/analysis/ArithmeticExpr.java

```diff
@@ -141,8 +141,26 @@ private static Type nonDecimalToDecimal(Type type) {
         }
     }
 
+    public static void getAddReturnTypeOfDecimal(TypeTriple triple, ScalarType lhsType, ScalarType rhsType) {
+        final int lhsPrecision = lhsType.getPrecision();
+        final int rhsPrecision = rhsType.getPrecision();
+        final int lhsScale = lhsType.getScalarScale();
+        final int rhsScale = rhsType.getScalarScale();
+
+        int delataScale = Math.abs(lhsScale - rhsScale);
+        int retPrecision = Math.max(lhsPrecision, rhsPrecision) + 1 + delataScale;
+        int retScale = Math.max(lhsScale, rhsScale);
+        // precision
+        retPrecision = Math.min(retPrecision, 38);
+        PrimitiveType decimalType = PrimitiveType.getDecimalPrimitiveType(retPrecision);
```

Comment:
decimal64(7,2) + decimal64(7,2) = decimal32(8,2), suggest to use decimal64, 
result type should use a primitive type not narrower than both lhs and rhs to avoid cast expr interplotation.